FirebaseInAppMessaging/Swift/Tests/Integration/FIAMSwiftUI/Podfile

```diff
@@ -0,0 +1,15 @@
+source 'https://github.com/firebase/SpecsStaging.git'
```

Comment:
Ideally there should be a build test in CI for the Integration test project, but should be ok to do later.